cypress/e2e/course/addBBBToolToCourseInNBC.feature

```diff
@@ -0,0 +1,54 @@
+@release
+Feature: Teacher can add and remove BBB tool in the course in NBC with admin permission
+
+  As a teacher I want to activate and deactivate BBB tool inside the course in NBC
+
+  @stable_test
+  Scenario Outline: Admin enables the video conference in the school settings page
+    Given I am logged in as a 'admin1_nbc' at 'nbc'
+    When I go to administration page
+    When I click on manage school card
+    #remove the following line if old admin page is hidden
+    When I click on general settings panel
+    Then I enable the video conference
+    Then I click on admin setting save button
+
+  @stable_test
+  Scenario Outline: Teacher can add and remove BBB tool in the course in NBC
+  #Teacher adds the BBB tool in the course
+    Given I am logged in as a 'teacher1_nbc' at 'nbc'
+    When I go to rooms overview
+    When I go to room 'German'
+    When I open course edit page
+    Then I can see course edit page
+    When I click on check box Activate video conferences in course edit page to enable the BBB tool
+    When I click on button Save changes in course edit page
```

Comment:
see above

Reply:
This step was already there and I reused it from the common course steps, but I updated/corrected it now.